rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -636,13 +636,7 @@
         "type": "datepicker",
         "templateOptions": {
           "hideLabel": true,
-          "wrappers": [
-            "toggle-switch"
-          ],
-          "toggle-switch": {
-            "label": "New acquisition",
-            "description": "If this option is enabled, this item is going to appear in the new acquisition list of the library."
-          }
+          "defaultValueExpression": "new Date().toISOString().split('T')[0]"
```

Comment:
No activation with a toggle switch any more?

Reply:
No, because it was an extra step. Now, as soon as we add the field, it is filled with the date.